test/fixtures/default/cisconx-64a

```diff
@@ -10,10 +10,10 @@ expected_hash:
       pe_upgrade_ver: 
       hypervisor: vmpooler
       platform: cisco_nexus-7-x86_64
-      template: cisco-nxos-9k-x86_64
       vrf: management
       ssh:
         user: beaker
+      template: cisco-nxos-9k-x86_64
```

Comment:
I suppose `rake generate:fixtures` wanted this in a different order?  This fixture was already here, it just reordered these lines when I ran the rake task.